src/java.base/share/classes/jdk/internal/loader/BuiltinClassLoader.java

```diff
@@ -242,16 +242,21 @@ private boolean usingSharedClasses() {
      * Returns if -Xshareclasses is used in the command line         							//IBM-shared_classes_misc
      */  																						//IBM-shared_classes_misc
     private static boolean isSharedClassesEnabled() {											//IBM-shared_classes_misc
-    	return AccessController.doPrivileged((PrivilegedAction<Boolean>) ()						//IBM-shared_classes_misc
-    			-> Boolean.getBoolean("com.ibm.oti.shared.enabled"));							//IBM-shared_classes_misc
+    	return AccessController.doPrivileged(new PrivilegedAction<Boolean>() {					//IBM-shared_classes_misc
```

Comment:
Please add a comment to the code explaining why a lambda can't be used here.

Reply:
Added comments, please review.